resources/views/bread/edit-add.blade.php

```diff
@@ -83,9 +83,31 @@ class="form-edit-add"
 
                         <div class="panel-footer">
                             <button type="submit" class="btn btn-primary save">{{ __('voyager::generic.save') }}</button>
+                            <button type="button" class="btn btn-danger js-delete">{{ __('voyager::generic.delete') }}</button>
```

Comment:
Again, not really a fan of the generic class naming `js-delete`.  Please come up with something more specific.